src/axom/slic/interface/slic.hpp

```diff
@@ -154,13 +155,28 @@ void setAbortFunction(AbortFunctionPtr abort_func);
  */
 void addStreamToMsgLevel(LogStream* ls, message::Level level);
 
+/*!
+ * \brief Adds the given GenericOutputStream to the the given level.
```

Comment:
Minor typo:
```suggestion
 * \brief Adds the given GenericOutputStream to the given level.
```

(The same typo appears a few lines up with the original function :)